src/Dapr.Actors.AspNetCore/DaprActorSetupFilter.cs

```diff
@@ -8,7 +8,6 @@ namespace Dapr.Actors.AspNetCore
     using System;
     using Microsoft.AspNetCore.Builder;
     using Microsoft.AspNetCore.Hosting;
-    using Microsoft.AspNetCore.Routing;
 
     internal class DaprActorSetupFilter : IStartupFilter
```

Comment:
Why do we still need a startup filter?

Reply:
To hook everything from the IWebHostBuilder extension.

Reply:
I mean I understand that but I'm asking why we need it at all? Is it because we want this model to treat ASP.NET Core like an implementation detail?

Reply:
yes

Reply:
I talked about this a bit offline with @amanbha today - right now they treat ASP.NET Core as an implementation detail, but still include `Startup` in examples.

- For someone who wants **just** actors and doesn't care about asp.net core - they should probably use the host builder and drop `Startup` from the example
- For someone who wants to use **actors and asp.net core** it makes sense for them to plug into routing directly, and for the app to have a real middleware pipeline that the user sees. 

Discussion about how to do that second one lead to this PR 😆